tests/parsing/errors/scanner/__snapshots__/parse.spec.js.snap

```diff
@@ -165,11 +165,11 @@ exports[`unclosedComment.js 1`] = `
 
 exports[`unclosedString.js 1`] = `
 "=====Parsetree==========================================
-let z = \\"eof\\"
+let z = \\"eof\\\\n\\"
 =====Errors=============================================
 
   Syntax error!
-  parsing/errors/scanner/unclosedString.js:1:10-2:0  
+  parsing/errors/scanner/unclosedString.js:1:9-2:0  
```

Comment:
@IwanKaramazow is there a case where we might need to highlight an ending `"`? I've only included the beginning `"`

Reply:
Hmm good question, my first reaction would be: unclosed strings will never have one. So we can't highlight it either way?